book.json

```diff
@@ -1,12 +1,28 @@
 {
   "title": "skatejs",
-  "plugins": ["jsfiddle"],
+  "plugins": [
+    "github",
+    "edit-link",
+    "anchors",
+    "jsfiddle"
```

Comment:
I was trying to see how this was presented to the user, but couldn't find a good explanation on the plugin itself. Is there a good example out there?